Lib/subprocess.py

```diff
@@ -71,6 +71,10 @@ class pywintypes:
 else:
     import select
     _has_poll = hasattr(select, 'poll')
+    try:
+        import threading
+    except ImportError:
+        threading = None
```

Comment:
Maybe use `dummy_threading`?

Reply:
dummy_threading doesn't give us anything useful.  this is fine.